plugins/action/ce.py

```diff
@@ -74,6 +78,9 @@ def run(self, tmp=None, task_vars=None):
                 # make sure a transport value is set in args
                 self._task.args['transport'] = transport
                 self._task.args['provider'] = provider
+                warnings.append(
+                    ['connection local support for this module is deprecated and will be removed after date 2022-09-25,'
+                     ' use connection %s' % pc.connection])
```

Comment:
1. community.network deprecates by version, not by date (see #81).
2. This does not belong into `warnings`, but into `deprecations` (https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L2182-L2184), with the format as described here: https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/common/warnings.py#L28